gensim/summarization/bm25.py

```diff
@@ -40,16 +40,17 @@ def initialize(self):
                 self.df[word] += 1
 
         for word, freq in iteritems(self.df):
-            self.idf[word] = math.log(self.corpus_size-freq+0.5) - math.log(freq+0.5)
+            self.idf[word] = math.log(self.corpus_size - freq + 0.5) - math.log(freq + 0.5)
 
     def get_score(self, document, index, average_idf):
         score = 0
         for word in document:
             if word not in self.f[index]:
                 continue
             idf = self.idf[word] if self.idf[word] >= 0 else EPSILON * average_idf
-            score += (idf*self.f[index][word]*(PARAM_K1+1)
-                      / (self.f[index][word] + PARAM_K1*(1 - PARAM_B+PARAM_B*self.corpus_size / self.avgdl)))
+            score += (
+                idf * self.f[index][word] * (PARAM_K1 + 1) /
+                (self.f[index][word] + PARAM_K1 * (1 - PARAM_B + PARAM_B * self.corpus_size / self.avgdl)))
```

Comment:
And just to confirm the hanging indents are applicable to multi-line return statements too?